flow/probes/ovssflow.go

```diff
@@ -183,12 +183,19 @@ func (o *OvsSFlowProbesHandler) RegisterProbeOnBridge(bridgeUUID string, tid str
 	opts := tableOptsFromCapture(capture)
 	ft := o.fpta.Alloc(tid, opts)
 
+	if capture.SamplingRate < 1 {
```

Comment:
With the current approach in order to get the counters we need to have samples(meaning flow) but I may want to not have flows. So I propose to do the following thing

```
if capture.SamplingRate < 1 {
   capture.SamplingRate = math.MaxUint32
}
```

In that case the number to so big that no sample will be sent and we will have the counters. What do you think ?

Reply:
Yes, we can but then we will not able to capture any flows using default capture type (ovssflow) on ovs-bridge. (Will get counters only.)

And yes we can also take PollingInterval = 0, for no counters!

Reply:
On the client side cli/ui you have set the default to 1 and 10 which fine as it was like this previously, meaning that without doing something specific the user will  get the same behavior. What I'm suggesting is to allow the user to set 0 flows or 0 counters. Since in order to get Counters we need to set `SamplingRate` to something > 0 I suggest to set it to something huge so that the counters will be sent without any sample packet.